Documentation/contributing/release/backports.rst

```diff
@@ -94,13 +113,8 @@ One-time Setup
       $ git remote add johndoe git@github.com:johndoe/cilium.git
       $ git remote add upstream https://github.com/cilium/cilium.git
 
-#. Make sure you have a GitHub developer access token with the ``public_repos``
-   ``workflow`` scopes available. You can do this directly from
-   https://github.com/settings/tokens or by opening GitHub and then navigating
-   to: User Profile -> Settings -> Developer Settings -> Personal access token
-   -> Generate new token.
```

Comment:
I think these instructions are still needed, the backporter still needs to prepare their Github access token right?

Reply:
Yes, they are still needed. I've just reordered the instructions. :)

Reply:
> Thanks for the PR! Curious why docker compose was chosen for building and running the container. Couldn't a simple `docker build` and `docker run` do the job?

That's what I started out with, but switched to docker compose in order to replace the heavy-weight COPY operation with volume mounting user's git repo inside the container.